integration-cli/docker_cli_run_unix_test.go

```diff
@@ -242,13 +242,6 @@ func (s *DockerSuite) TestRunEchoStdoutWithCPUSharesAndMemoryLimit(c *check.C) {
 	}
 }
 
-func (s *DockerSuite) TestRunWithCpuset(c *check.C) {
```

Comment:
Yea, if the intent was that the deprecate docs were supposed to say `--cpuset` instead of `-cpuset` then let's fix that.  But let's be extra sure because the typo might be it being in the list at all, not the missing `-`.